relay/cmd/main.go

```diff
@@ -54,4 +80,6 @@ func main() {
 	if err != nil {
 		panic(fmt.Sprintf("fatal error starting relay server: %s", err))
 	}
+
+	return nil
```

Comment:
If this function now returns an error, should we convert the `panic()` operations in this function into errors?